File: java_SDK/PayHubSDK/src/main/java/com/payhub/ws/model/VerifyResponse.java
package com.payhub.ws.model;

import com.fasterxml.jackson.annotation.JsonInclude;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class VerifyResponse {
    
    private String verifyId;
    
    private String approvalCode;
    
    private String processedDateTime;
    
    private String avsResultCode;
    
    private String verificationResultCode;
    
    private String responseCode;
    
    private String responseText;
    
    private String cisNote;
    
    private String riskStatusResponseText;
    
    private String riskStatusRespondeCode;
    
    private String saleDateTime;
    
    private String tokenizedCard;
    
    private CustomerReference customerReference;

	public String getVerifyId() {
		return verifyId;
	}

	public void setVerifyId(String verifyId) {
		this.verifyId = verifyId;
	}

	public String getApprovalCode() {
		return approvalCode;
	}

	public void setApprovalCode(String approvalCode) {
		this.approvalCode = approvalCode;
	}

	public String getProcessedDateTime() {
		return processedDateTime;
	}

	public void setProcessedDateTime(String processedDateTime) {
		this.processedDateTime = processedDateTime;
	}

	public String getAvsResultCode() {
		return avsResultCode;
	}

	public void setAvsResultCode(String avsResultCode) {
		this.avsResultCode = avsResultCode;
	}

	public String getVerificationResultCode() {
		return verificationResultCode;
	}

	public void setVerificationResultCode(String verificationResultCode) {
		this.verificationResultCode = verificationResultCode;
	}

	public String getResponseCode() {
		return responseCode;
	}

	public void setResponseCode(String responseCode) {
		this.responseCode = responseCode;
	}

	public String getResponseText() {
		return responseText;
	}

	public void setResponseText(String responseText) {
		this.responseText = responseText;
	}

	public String getCisNote() {
		return cisNote;
	}

	public void setCisNote(String cisNote) {
		this.cisNote = cisNote;
	}

	public String getRiskStatusResponseText() {
		return riskStatusResponseText;
	}

	public void setRiskStatusResponseText(String riskStatusResponseText) {
		this.riskStatusResponseText = riskStatusResponseText;
	}

	public String getRiskStatusRespondeCode() {
		return riskStatusRespondeCode;
	}

	public void setRiskStatusRespondeCode(String riskStatusRespondeCode) {
		this.riskStatusRespondeCode = riskStatusRespondeCode;
	}

	public String getSaleDateTime() {
		return saleDateTime;
	}

	public void setSaleDateTime(String saleDateTime) {
		this.saleDateTime = saleDateTime;
	}

	public String getTokenizedCard() {
		return tokenizedCard;
	}

	public void setTokenizedCard(String tokenizedCard) {
		this.tokenizedCard = tokenizedCard;
	}

	public CustomerReference getCustomerReference() {
		return customerReference;
	}

	public void setCustomerReference(CustomerReference customerReference) {
		this.customerReference = customerReference;
	}
    
    
}

File: Payhub - SDK/php_SDK/com/payhub/ws/model/VerifyResponse.php
<?php

/**
 * Created by PhpStorm.
 * User: agustin
 * Date: 24/07/2015
 * Time: 11:13
 */
class VerifyResponse
{
    public $verifyId;
    public $approvalCode;
    public $processedDateTime;
    public $avsResultCode;
    public $verificationResultCode;
    public $responseCode;
    public $responseText;
    public $cisNote;
    public $riskStatusResponseText;
    public $riskStatusRespondeCode;
    public $saleDateTime;
    public $tokenizedCard;
    public $customerReference;

    /**
     * @return mixed
     */
    public function getVerifyId()
    {
        return $this->verifyId;
    }

    /**
     * @return mixed
     */
    public function getApprovalCode()
    {
        return $this->approvalCode;
    }

    /**
     * @return mixed
     */
    public function getProcessedDateTime()
    {
        return $this->processedDateTime;
    }

    /**
     * @return mixed
     */
    public function getAvsResultCode()
    {
        return $this->avsResultCode;
    }

    /**
     * @return mixed
     */
    public function getVerificationResultCode()
    {
        return $this->verificationResultCode;
    }

    /**
     * @return mixed
     */
    public function getResponseCode()
    {
        return $this->responseCode;
    }

    /**
     * @return mixed
     */
    public function getResponseText()
    {
        return $this->responseText;
    }

    /**
     * @return mixed
     */
    public function getCisNote()
    {
        return $this->cisNote;
    }

    /**
     * @return mixed
     */
    public function getRiskStatusResponseText()
    {
        return $this->riskStatusResponseText;
    }

    /**
     * @return mixed
     */
    public function getRiskStatusRespondeCode()
    {
        return $this->riskStatusRespondeCode;
    }

    /**
     * @return mixed
     */
    public function getSaleDateTime()
    {
        return $this->saleDateTime;
    }

    /**
     * @return mixed
     */
    public function getTokenizedCard()
    {
        return $this->tokenizedCard;
    }

    /**
     * @return mixed
     */
    public function getCustomerReference()
    {
        return $this->customerReference;
    }

    public static function fromArray($data){
        if(!is_null($data)) {
            $verif = new VerifyResponse();
            foreach ($data as $key => $value) {
                if (property_exists(get_class($verif), $key)) {
                    if ($key == "customerReference") {
                        $verif->{$key} = $value;
                    }else{
                        $verif->{$key} = $value;
                    }
                }
            }
            return $verif;
        }
    }
}